data-modeling/delegate-types/README.md

```diff
@@ -0,0 +1,68 @@
+# Data modeling: Delegate types
+
+This example shows you how to model [delegate types](https://edgeapi.rubyonrails.org/classes/ActiveRecord/DelegatedType.html) using Prisma, and use [Prisma Client](https://www.prisma.io/client) in a **TypeScript script** to read and write data.
+
+This example consists of a `script.ts` file that contains example Prisma Client queries for delegate types
+
+
+## What are Delegate types?
+
+An application such as LinkedIn contains a feed that would contain different types of content such as articles, images, and videos. In such an instance, you would have a *parent entity*, `Feed`, containing all the shared properties and *child entities*,`Article`, `Image`, and `Video`, that inherit the *parent entity*'s properties and have their individual properties. 
+
+When querying feed data, you would only query the feed data instead of querying the `Article`, `Image`, and `Video` tables and then merging them into one response. Querying and merging responses from the individual tables wouldn't allow you to perform operations such as ordering and filtering. This is the problem that delegate type solves.
```

Comment:
> When querying feed data, you would only query the feed data

Can you rephrase this to make this more clear?

Reply:
> When querying the feed, you would make a query only to the `Feed` table instead of querying...

Is this clearer?